schema/GeneticVariant_GenVarSource_enums.mcf

```diff
@@ -0,0 +1,129 @@
+# Generated by script format_dbSNP_GenVarSource_enum_schema.py
+
+Node: dcid:GenVarSourceKorea1k
+name: "Korea1k"
+typeOf: dcs:GenVarSourceEnum
```

Comment:
Should we add these to the existing GenomeAnnotationEnum.mcf file?

http://google3/third_party/datacommons/schema/GenomeAnnotationEnum.mcf?l=935-938&rcl=324872035

This PR is fine. But, for future schema changes, it might be simpler to make them in google3.

Reply:
These were generated by a script. I thought that they should be separate like we did for Mengzhen's dataset. That way when someone runs the script only this file will be effected.

Ok, I can submit the changes to google3 next time. I thought we were trying to shift to using GitHub now.